pagerduty/event_orchestration_integration.go

```diff
@@ -0,0 +1,143 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+)
+
+type EventOrchestrationIntegrationService service
+
+type EventOrchestrationIntegrationParameters struct {
+	RoutingKey string `json:"routing_key,omitempty"`
+	Type       string `json:"type,omitempty"`
+}
+
+type EventOrchestrationIntegration struct {
+	ID         string                                   `json:"id,omitempty"`
+	Label      string                                   `json:"label,omitempty"`
+	Parameters *EventOrchestrationIntegrationParameters `json:"parameters,omitempty"`
+}
+
+type EventOrchestrationIntegrationPayload struct {
+	Integration *EventOrchestrationIntegration `json:"integration,omitempty"`
+}
+
+type EventOrchestrationIntegrationMigrationPayload struct {
+	SourceType    string `json:"source_type,omitempty"`
+	SourceId      string `json:"source_id,omitempty"`
+	IntegrationId string `json:"integration_id,omitempty"`
+}
+
+type ListEventOrchestrationIntegrationsResponse struct {
+	Total        int                              `json:"total,omitempty"`
+	Integrations []*EventOrchestrationIntegration `json:"integrations,omitempty"`
+}
+
+func buildEventOrchestrationIntegrationUrl(orchestrationId string, lastUrlSegment string) string {
+	url := fmt.Sprintf("%s/%s/integrations", eventOrchestrationBaseUrl, orchestrationId)
+
+	if len(lastUrlSegment) > 0 {
+		url = fmt.Sprintf("%s/%s", url, lastUrlSegment)
+	}
+
+	return url
+}
+
+func (s *EventOrchestrationIntegrationService) List(orchestrationId string) (*ListEventOrchestrationIntegrationsResponse, *Response, error) {
+	return s.ListContext(context.Background(), orchestrationId)
+}
```

Comment:
TODO: delete non-context methods

Reply:
Addressed in [71ad28f](https://github.com/heimweh/go-pagerduty/pull/118/commits/71ad28f8687d2d144017913fe270824f0d634e9b)